src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -172,6 +178,8 @@ public function __construct($strTable)
 		$this->strTable = $strTable;
 		$this->blnIsDbAssisted = $GLOBALS['TL_DCA'][$strTable]['config']['databaseAssisted'];
 
+		$this->initPicker();
```

Comment:
I think this needs a similar check as [`DC_Table` has on line 202](https://github.com/leofeyer/core-bundle/blob/21ff0a8e9cee252147ce5d934d17bf35f94c7e63/src/Resources/contao/drivers/DC_Table.php#L202). Currently this line leads to a `The table name must not be empty` exception if I open `/contao?do=files`.